app/src/main/java/com/iron/espresso/presentation/home/mystudy/StudyDetailActivity.kt

```diff
@@ -72,7 +71,10 @@ class StudyDetailActivity :
 
     companion object {
         private const val TOOLBAR_TITLE = "title"
-        fun getInstance(context: Context, title: String) =
-            Intent(context, StudyDetailActivity::class.java).putExtra(TOOLBAR_TITLE, title)
+        private const val STUDY_ID = "studyId"
+        fun getInstance(context: Context, title: String?, id: Int?) =
```

Comment:
굳이 nullable로 받을 필요 있는 부분일지?
```suggestion
        fun getInstance(context: Context, title: String, id: Int) =
```

Reply:
faa16b5 적용했습니다! @wswon